cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1282,6 +1284,9 @@ private List<ContainerPort> getContainerPortList(KafkaPool pool) {
         if (metrics.isEnabled()) {
             ports.add(ContainerUtils.createContainerPort(MetricsModel.METRICS_PORT_NAME, MetricsModel.METRICS_PORT));
         }
+        else if (strimziMetrics.isEnabled()) {
+            ports.add(ContainerUtils.createContainerPort(StrimziReporterMetricsModel.METRICS_PORT_NAME, StrimziReporterMetricsModel.METRICS_PORT));
+        }
```

Comment:
I think we should have some abstract class / interface for both metric types that will be then separated by the type. That would allow you to define the port there. But the port should in any case be the same. So it should use a central definition.